docs/guides/reference/ard-processing-extent.md

```diff
@@ -0,0 +1,93 @@
+# DEA ARD processing extent
+
+## Background

```

Comment:
@OHehir-Aaron Can I delete this 'Background' heading so that effectively this section becomes the 'introduction' section.

Reply:
Yep!